addons/web_editor/static/lib/odoo-editor/test/editor-test.js

```diff
@@ -25,6 +25,35 @@ mocha.run(failures => {
     } else {
         console.log('test successful');
     }
+
+    // Clean report of successful tests to ease DEV / debug
+    // just add ?clean=1
     const reportEl = document.getElementById('mocha-report');
     window.scrollTo(0, window.scrollY + reportEl.getBoundingClientRect().top);
+    const urlParams = new URLSearchParams(window.location.search);
+    if (urlParams.get('clean')) {
+        console.log('Successful test have been removed.');
+        const report = document.querySelector("#mocha-report");
+
+        let hr = document.createElement("hr");
+        report.prepend(hr);
+        let h1 = document.createElement("h1");
+        h1.textContent = "Clean : Successfull tests have been removed.";
+        report.prepend(h1);
+
+        const passedTests = report.querySelectorAll('.test.pass');
+        for (let el of passedTests) {
+            el.remove();
+        }
+        const cleanUl = function () {
+            const emptyUl = report.querySelectorAll('ul:empty');
+            for (let ul of emptyUl) {
+                ul.parentElement.remove();
+            }
+        };
+        cleanUl();
+        cleanUl();
+        cleanUl();
+        cleanUl();
```

Comment:
Because the UL are multiple and nested in the test report, so I call the method multiple time to clean all unnecessary title. It's crude but is not really a critical feature so I did not bother make it look cleaner.